File: Mahasiswa.cs
namespace KKN_Nagrak_RT04
{
    public class Mahasiswa
    {
        public string Nama { get; set; }
        public string Jabatan { get; set; }
        public string Prodi { get; set; }

        public Mahasiswa(string nama, string jabatan, string prodi)
        {
            Nama = nama;
            Jabatan = jabatan;
            Prodi = prodi;
        }
    }
}

File: mahasiswa.java
public class Mahasiswa {
    private String nama;
    private String jabatan;
    private String prodi;

    public Mahasiswa(String nama, String jabatan, String prodi) {
        this.nama = nama;
        this.jabatan = jabatan;
        this.prodi = prodi;
    }

    public String getNama() { return nama; }
    public String getJabatan() { return jabatan; }
    public String getProdi() { return prodi; }
}
